testfiles/code0.txt

```diff
@@ -0,0 +1,24 @@
+
+function test_3 returns int with args int a, int temp, float test
+
+initialise b=10
+test=10
+a = b/test
+return a
+endfunction
+
+start
+initialise hi=20
+initialise int a
+input a
```

Comment:
I feel it is better to add a print statement before executing the scanf cause user doesn't know that he has to give an input. And we don't explicitly mention that in sudo code but I feel it is better to do it.